lisa/nic.py

```diff
@@ -235,22 +237,23 @@ def _get_node_nic_info(self) -> None:
         # use sysfs to gather upper/lower nic pairings and pci slot info
         nic_info_fetch_cmd = "ls -la /sys/class/net/*/lower*/device"
         self._node.log.info(f"Gathering NIC information on {self._node.name}.")
-        result = self._node.execute(
-            nic_info_fetch_cmd,
-            shell=True,
-            expected_exit_code=0,
-            expected_exit_code_failure_message="Could not grab NIC device info.",
-        )
-        for line in result.stdout.splitlines():
-            match = self.__nic_lower_regex.search(line)
-            if not match:
-                raise LisaException(
-                    "Could not locate default network interface"
-                    f" in output:\n{result.stdout}"
-                )
-            upper_nic, lower_nic, pci_slot = match.groups()
-            nic_info = NicInfo(upper_nic, lower_nic, pci_slot)
-            self.append(nic_info)
+        result = self._node.execute(nic_info_fetch_cmd, shell=True)
+        if self.__file_not_exist.match(result.stdout):
+            self._node.log.debug(f"There is no VF on node {self._node.name}.")
```

Comment:
Do we need this log line? It looks not provide many information.

Reply:
removed